tests/unit/plugins/module_utils/mock_info_api.py

```diff
@@ -1636,6 +1636,41 @@ def get_filesystem_list(response_type):
         else:
             return "Failed to get details of Filesystem"
 
+    @staticmethod
+    def get_writable_snapshot_list(response_type):
+        if response_type == 'api':
+            return {
```

Comment:
if data is same then, I guess if-else block is not required

Reply:
We have a difference of a key "writable" in first case.